src/valuetypes.ts

```diff
@@ -435,15 +443,21 @@ export class ValueTypeEditor implements IValueTypeEditor {
     return this.impl.parse(def, data, accessor);
   }
 
-  guessOptions(def: ITypeDefinition, data: any[], accessor: (row: any) => any) {
+  async guessOptions(def: ITypeDefinition, data: any[], accessor: (row: any) => any) {
     def.type = this.id;
-    return this.impl.guessOptions(def, data, accessor);
+    const types = await this.impl.guessOptions(def, data, accessor);
+    this.desc.subType = types.subType;
```

Comment:
I wanted to save the subtype for IDTypes (Ensembl, ...) that I don't always have to call guessOptions to get the detected IDType, but this was more work in progress
I thought of either calling the guessOptions method with the context bound to it to save it to `this` or saving it to the config in  ordino_uploaded_data/src/uploader.ts around line 132 and passing it to the createTypeEditor method.
Which one is preferrable? Or do you have a cleaner solution?